### DISCUSSION TITLE
Web UI - Maybe open source could be a faster way to production?

### DISCUSSION THREAD

**Discussion by dsoundmn** [rustdesk/rustdesk#4170] [General]
### Description

Open source the Web UI / authentication platform

### Impact

More usability and higher adoption - plus less forking of projects and better OSS contributions

### Additional Context

Hi Folks - I certainly appreciate the idea of using this as a leverage point for commercial services - but I also bet that not only would you be able to have a complete interface with OAuth or LDAP authentication (office 365, google, etc...) but that you would be able to make just that much more in your offering hosted services. Either way - you've made a great application that would benefit most from not pushing developers to fork their own API/web server and instead have us all work together to improve it. Personally, I want to integrate OAuth for Google Workspace / MS 365 to start with - maybe even Okta and having to start a system from scratch is just daunting and counterproductive when there is a foundation of some sort.

Thank you!

**Comment by cwchristerw** [comment]
I'm interested in getting OpenID Connect support. When I test new programs I require that I can integrate it to my existing authentication and authorization workflow. In the past I have had multiple accounts in different systems but when you have over 10 different applications its better to just use e.g. Keycloak for authentication and authorization.

I currently use Keycloak for OpenID Connect and FreeIPA for LDAP and Kerberos.
